src/ImageMap/index.tsx

```diff
@@ -0,0 +1,54 @@
+import { Spin } from 'antd';
+import * as React from 'react';
+import { useImageSize } from 'react-image-size';
+
+import { MllSpinnerIcon } from '../Spinner';
+import { notification } from '../notification';
+
+export { PopOverArea } from './PopOverArea';
+export { HiddenArea } from './HiddenArea';
+
+export type ImageMapProps = {
+  src: string;
+  children: React.ReactElement;
+};
+
+export function ImageMap(props: ImageMapProps) {
+  const [dimensions, { loading, error }] = useImageSize(props.src);
+
+  if (error) {
+    notification.error({
+      message: error,
+    });
+  }
```

Comment:
I think an optional `onError` callback would be more flexible, e.g. allowing to control positioning, duration, etc.